src/test/java/org/b612foundation/adam/datamodel/estimation/OorbConfigWriterTest.java

```diff
@@ -0,0 +1,31 @@
+package org.b612foundation.adam.datamodel.estimation;
+
+import org.b612foundation.adam.datamodel.PropagationConfigurationFactory;
+import org.b612foundation.adam.datamodel.PropagatorConfiguration;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class OorbConfigWriterTest {
+
+  @Test
+  public void testDefaultConfigFile() throws IOException {
+    var config = OorbEstimateConfigurationFactory.buildDefaultConfig();
```

Comment:
pardon the ignorance: what is it about the default config that you're testing here?

Reply:
Just testing that it's correctly serializing/deserializing at all. At the time it was also being used for dumping some default JSON we could then hand edit. We can probably take it out at this point.

Reply:
Ah ok, np. I was initially confused why there weren't any assertions.